release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3116,6 +3116,68 @@ $ ccoctl azure create-all \
 --
 (link:https://issues.redhat.com/browse/OCPBUGS-22651[*OCPBUGS-22651*])
 
+* When installing an {product-title} cluster with static IP addressing and Tang encryption, worker nodes boot without network settings. This condition prevents worker nodes from accessing the Tang server, causing installation to fail. To address this condition, you must set the network settings for each worker node as `ip` kernel arguments.
++
+When using installer-provisioned installation, provide `ip` kernel arguments for each worker node using a manifest. For example: 
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  name: 99-worker-tang-network-config
+  labels:
+    machineconfiguration.openshift.io/role: worker
+spec:
+  config:
+    ignition:
+      version: 3.1.0
+  kernelArguments:
+    - 'ip=<static-ip>::<gateway>:<netmask>:<hostname-1>:<interface>:none'
+    - 'ip=<static-ip>::<gateway>:<netmask>:<hostname-2>:<interface>:none'
+    - 'rd.neednet=1'
+----
++
+Save and copy the manifest file to the `clusterconfigs/openshift` directory:
++
+[source,terminal]
+----
+$ cp ~/router-replicas.yaml clusterconfigs/openshift/99-worker-tang-network-config.yaml
+----
++
+Apply the manifest: 
++
+[source,terminal]
+----
+$ oc apply -f clusterconfigs/openshift/99-worker-tang-network-config.yaml
+----
++
+When installing with the {ai-full}, modify the infrastructure environment using the API to append the network settings as `ip` kernel arguments for each worker node. For example:
++
+[source,bash]
+----
+$ curl https://api.openshift.com/api/assisted-install/v2/infra-envs/${INFRA_ENV_ID} \
+-X PATCH \
+-H "Authorization: Bearer ${API_TOKEN}" \
+-H "Content-Type: application/json" \
+-d "$(jq --null-input \
+  --slurpfile pull_secret ~/Downloads/pull-secret.txt '
+    {
+      "kernel_arguments": [
+        { "operation": "append", "value": "ip=<static-ip>::<gateway>:<netmask>:<hostname-1>:<interface>:none" },
+        { "operation": "append", "value": "ip=<static-ip>::<gateway>:<netmask>:<hostname-2>:<interface>:none" },
+        { "operation": "append", "value": "rd.neednet=1" }
+      ],
+      "image_type":"minimal-iso",
+      "pull_secret": $pull_secret[0] | tojson
+    }
+')" | jq
+----
++
+For the previous examples, replace `<static-ip>` with the static IP address for the node, for example, `192.168.1.100`; `<gateway>` with the IP address of your network's gateway, for example, `192.168.1.1`; `<netmask>` with the network mask, for example, `255.255.255.0`; `<hostname-1>` and `<hostname-2>` with the desired hostnames for the nodes, for example, `node1.example.com`; and, `<interface>` with the name of the network interface, for example, `eth0`.
```

Comment:
I would suggest to consider using the <1>, <2>, .... <n> callouts to represent this information.

Reply:
You don't need to have one callout per variable. Bullets work in a callout, for example, if you enclose the lines in a block.